libraries/ui-library/src/components/six-breadcrumbs-item/six-breadcrumbs-item.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, Event, EventEmitter, h, Prop } from '@stencil/core';
+import { EmptyPayload } from '../../utils/types';
+
+@Component({
+  tag: 'six-breadcrumbs-item',
+  styleUrl: 'six-breadcrumbs-item.scss',
+  shadow: true,
+})
+export class SixBreadcrumbsItem {
+  /** Name or label of the breadcrumb */
+  @Prop() name: string = '';
+
+  /** If the breadcrumb is disabled */
+  @Prop() disabled: boolean = false;
+
+  /** Emitted on click. */
+  @Event({ eventName: 'sixClick' }) sixClickEvent!: EventEmitter<EmptyPayload>;
```

Comment:
In the other components we use kebab case like this six-input-change. 

I think there was a reason for this, maybe @ibirrer you remember why we changed that? 🤔

Reply:
I wasn’t part of the team when this decision was made. I believe Stencil recommends camelCase, but since all the others use kebab-case, I recommend changing it to kebab-case here as well.